spec/requests/stories_index_spec.rb

```diff
@@ -26,6 +26,12 @@
       renders_proper_sidebar(navigation_link)
     end
 
+    it "doesn't render a featured scheduled article" do
+      article = create(:article, featured: true, published_at: 1.hour.from_now)
```

Comment:
Reading notes: I love the specificity of this; it's helpful to understand the implications of future published at.